moped-database/migrations/1691605248601_council-district-trigger/up.sql

```diff
@@ -0,0 +1,157 @@
+-- add spatial index to this layer
+CREATE INDEX layer_council_district_geom_idx
+  ON layer_council_district
+  USING GIST (geography);
+
+-- create table to hold feature-district associations
+CREATE TABLE public.features_council_districts (
+    id serial PRIMARY KEY,
+    feature_id integer NOT NULL,
+    council_district_id integer NOT NULL,
+    FOREIGN KEY ("council_district_id") REFERENCES "public"."layer_council_district" ("council_district")
+        ON UPDATE CASCADE ON DELETE CASCADE
+);
```

Comment:
note that we cannot establish a foreign key constrain from this table's `feature_id` column to the `features.id` column. This is a [limitation of using table inheritance](https://www.postgresql.org/docs/current/ddl-inherit.html#DDL-INHERIT-CAVEATS).